sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -2152,6 +2231,23 @@ export interface SearchSuggester {
     sourceFields: string[];
 }
 
+// @public
+export interface SemanticConfiguration {
+    name: string;
+    prioritizedFields: PrioritizedFields;
+}
+
+// @public
+export interface SemanticField {
+    // (undocumented)
+    fieldName?: string;
```

Comment:
nit: since this is a `SementicField` do we need `field` in the property name? Could this be simply `name` ?

Reply:
Done